### PR TITLE
Custom exception handling for Authentication and User services

### PR DESCRIPTION
**Pull request by nmamro**

### CODE REVIEW

File: Common/Kernel/CustomExceptionHandler.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+{
+    public static class CustomExceptionHandler
+    {
+        public static async Task HandleCustomException(HttpContext context)
+        {
+            var handler = context.Features.Get<IExceptionHandlerPathFeature>();
+
+            var response = new ErrorResponse
+            {
+                UtcTime = DateTime.Now
```
Comment (studokim) [inline Common/Kernel/CustomExceptionHandler.cs]: I suppose, `DateTime.UtcNow`

Comment (nmamro) [reply Common/Kernel/CustomExceptionHandler.cs]: > I suppose, `DateTime.UtcNow`

Thanks, fixing in next commit

**Comment by nmamro** [comment]
I don't mention project in branch and pr name because technically there's also injection in services, not just delegate in Kernel.

**Review by studokim — APPROVED** [review]
Maybe remove some empty lines

File: Common/Kernel/CustomExceptionHandler.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+using DTO;
+using Microsoft.AspNetCore.Diagnostics;
+using System.Text.Json;
+using Kernel.CustomExceptions;
+
+namespace Kernel
+{
+    public static class CustomExceptionHandler
+    {
+        public static async Task HandleCustomException(HttpContext context)
```
Comment (DarkFelas) [inline Common/Kernel/CustomExceptionHandler.cs:13]: Rename in Handle(). It is already static class CustomExceptionHandler and there is some duplications. CustomExceptionHandler.HandleCustomException()

**Review by strannick-rodion — APPROVED** [review]

**Review by strannick-rodion — APPROVED** [review]

**Review by Cerberous007 — APPROVED** [review]
Don`t forget to follow the code convention in title of PR (CamelCase, Where_What etc)

**Review by ifelseelif — APPROVED** [review]

**Pull request merged**